Jenny- testing pull request with FD navbar

JRRFranklin opened the pull request:
I'm mainly creating this request (as opposed to just committing at this point) so I can check that I'm doing it correctly. We don't have to approve and merge, but I want to make sure that I'm following the right steps should I be happy with the commits.

JRRFranklin committed:
Changed .jsp to .html

JRRFranklin committed:
Added first draft navbar

simple black first draft nav-bar. I'm only linking to Semantic at this point, so I can't edit colors, etc. yet.

jameswillhoite commented on src/main/webapp/index.html:
```diff
@@ -0,0 +1,59 @@
+            crossorigin="anonymous">
+    </script>
+
+    <script src="semantic/dist/semantic.min.js"></script>
```
This is a relative link, is this going to be hosted on the server?

JRRFranklin replied:
Sorry, I didn't see this. I need to ask about that, as I think it should be in a folder on the server. I'm currently a little foggy on exactly what I'm calling, but it works (?) When I figure that out, I'm going to download it so I can go in and edit where needed. Does that answer your question?

jameswillhoite replied:
Everything else looks good on this code. Just this semantic.min.js script. It would be faster to host on the server, but it can be a cdn.

jameswillhoite reviewed (changes requested):
I don't see the semantic.min.js as part of the commit? Will it be hosted in this path or should it be a CDN?

JRRFranklin committed:
Added a first draft footer
Added a place-holder logo in a "image" folder I created

JRRFranklin committed:
added place-holder hero-image

Semantic doens't have a built in hero image feature. When I download semantic to edit the css myself, I can fix this. So for now, I've added a placeholder image with no text.

jameswillhoite reviewed (approved):
besides adding the semantic.min.js to the server everything else looks okay

JRRFranklin committed:
added feature boxes and bug report btn

JRRFranklin committed:
Merge branch 'master' into jenny-index-page
